hw/drivers/sensors/lis2dw12/include/lis2dw12/lis2dw12.h

```diff
@@ -579,6 +579,26 @@ lis2dw12_set_int1_pin_cfg(struct sensor_itf *itf, uint8_t cfg);
  */
 int lis2dw12_set_int2_pin_cfg(struct sensor_itf *itf, uint8_t cfg);
 
+/**
+ * Clear interrupt pin configuration for interrupt 1
+ *
+ * @param the sensor interface
```

Comment:
`itf` is missing from this `@param` comment (same for `lis2dw12_clear_int1_pin_cfg`).

Reply:
Do you mean the `itf` text in the comments ? None of my comments have that. Is it needed ?